tests/test_cut_clip.py

```diff
@@ -387,8 +387,20 @@ def test_in_out_values(self):
         self.assertEqual(clip.tail_in.to_frames(), 20000 + 9 + 1)
         self.assertEqual(clip.tail_out.to_frames(), clip.tail_in.to_frames() + sg_settings.default_tail_duration - 1)
 
-        # Kevin: sg_settings.timecode_in_to_frame_mapping_mode = _TC2FRAME_AUTOMATIC_MODE
-        # Value from a SG Cut Item should be used as a base for an offset
+        # Switch to AUTOMATIC mode, and make sure the values are recomputed
+        # correctly. Do not set a cut item for now.
+        sg_settings.timecode_in_to_frame_mapping_mode = _TC2FRAME_AUTOMATIC_MODE
+        clip.sg_shot = None  # Unsetting the Shot forces a recompute
+        self.assertEqual(clip.compute_cut_in().to_frames(), 1009)
```

Comment:
It would be better to set the `default_head_duration` to an unusual value (like 666), and then compare values against the settings:
- head in is the default value.
- cut in is head in + default handles duration
- cut out is cut in + clip duration
- tail in is cut out + 1
- tail out is tail in + default handles duration -1